apps/files_trashbin/lib/trashbin.php

```
@@ -62,7 +62,11 @@ class Trashbin {
 	 * @param array $params
 	 */
 	public static function ensureFileScannedHook($params) {
-		self::getUidAndFilename($params['path']);
+		try {
+			self::getUidAndFilename($params['path']);
+		} catch (NotFoundException $e) {
+
```

ewww, log the error? Or at least add a comment why it is ok to ignore the error.